modules/theme/src/themes/default/modules/modal.overrides

```diff
@@ -278,3 +278,32 @@
         }
     }
 }
+
+/*********************************************
+                    TOTP
+*********************************************/
+
+.ui.modal {
+    .totp-header {
+        height: 150px;
+        background-color: @primaryColor;
+        color: @white;
+        border: none;
+        padding: 0;
+        display: flex;
+        justify-content: center;
+        align-items: end;
+
+        .illustration {
+            width: 150px;
+            height: 150px;
+            overflow: hidden;
+        }
+    }
+
+    .link {
```

Comment:
shall we add a wrapper class to the totp modal. Since this approach will override styles for elements with `.link` class inside a modal.